components/develop/modules/ROOT/nav.adoc

```diff
@@ -4,8 +4,15 @@
 * Network Details
 ** xref:hackathon.adoc[Hackathon Notes]
 ** xref:networks.adoc[Networks]
-** xref:sfuel-gas-token.adoc[SKALE Gas Token]
 ** xref:proof-of-work.adoc[Proof of Work]
```

Comment:
** xref:proof-of-work.adoc[Proof of Work] should be moved below

Reply:
Resolved by [commit 280fa47](https://github.com/TheGreatAxios/docs.skale.network/commit/280fa47e079d8a2822001dd96462d5705142091d)

Included redirect